src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -265,7 +265,6 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 					if (originalImageName !== attachmentName) {
 						imageCalloutResult.insertEscapedMarkdown = `![${attachmentName}](attachment:${attachmentName.replace(/\s/g, '')})`;
 					}
-					await insertFormattedMarkdown(imageCalloutResult.insertEscapedMarkdown, this.getCellEditorControl());
```

Comment:
Was this just a bad merge then from what you can tell?

Reply:
Huh, guess not. https://github.com/microsoft/azuredatastudio/pull/15238 is where it was introduced (3 years ago)...

Were you able to figure out what changed to cause this to break then?

Reply:
Hey Charles, yeah I'm still not sure what change from the vscode merge caused the issue, but the getValue() function wasn't working properly before. I tried debugging it and the first time is called it returns an empty string and after the merge it returns the actual value in the editor. I think that's why we added the insertFormattedMarkdown twice when first implementing it.
https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/contrib/notebook/browser/cellViews/code.component.ts#L254